Code/GraphMol/FileParsers/file_parsers_catch.cpp

```diff
@@ -4593,4 +4593,40 @@ M  END)CTAB";
       CHECK(m->getBondWithIdx(0)->getIsAromatic());
     }
   }
-}
\ No newline at end of file
+}
+
+TEST_CASE(
+    "Github #5165: issue with V3000 SD files containing enhanced "
+    "stereochemistry information") {
+    
+    SECTION("as reported") {
+        std::string rdbase = getenv("RDBASE");
+        std::string fName =
+          rdbase + "/Code/GraphMol/FileParsers/test_data/mol_with_enhanced_stereo_2_And_groups.sdf";
+        SDMolSupplier suppl(fName);
+        ROMol *mol = suppl.next();
+        REQUIRE(mol);
+        auto groups = mol->getStereoGroups();
+        CHECK(groups.size()==2);
+        if (groups.size()==2) {
+            CHECK(groups[0].getGroupType() == RDKit::StereoGroupType::STEREO_AND);
+            CHECK(groups[1].getGroupType() == RDKit::StereoGroupType::STEREO_AND);
+        }
```

Comment:
Would this be functionally the same? It seems like it would make for a bit cleaner code
```suggestion
        REQUIRE(groups.size()==2);
        CHECK(groups[0].getGroupType() == RDKit::StereoGroupType::STEREO_AND);
        CHECK(groups[1].getGroupType() == RDKit::StereoGroupType::STEREO_AND);
```

Reply:
Yeah, I think the second form is even better. Nice thought!

Reply:
Ok thanks!